nifi-nar-bundles/nifi-influxdb-bundle/nifi-influxdb-processors/pom.xml

```diff
@@ -0,0 +1,88 @@
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License. You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.nifi</groupId>
+        <artifactId>nifi-influxdb-bundle</artifactId>
+        <version>1.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>nifi-influxdb-processors</artifactId>
+    <packaging>jar</packaging>
+
+    <dependencies>
+	    <dependency>
+	        <groupId>org.influxdb</groupId>
+  	        <artifactId>influxdb-java</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-utils</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-mock</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/IT*.java</exclude>
```

Comment:
I'm not sure this is really needed as we're already defining what tests have to be included in the root pom XML file.
````xml
<includes>
    <include>**/*Test.class</include>
    <include>**/Test*.class</include>
    <include>**/*Spec.class</include>
</includes>
````

Reply:
@pvillard31 

If I remove this profile/configuration from the pom.xml and run `mvn clean install` in that directory, the integration tests get executed and fail if InfluxDb is not running on the local server.  

```
Tests in error: 
  ITPutInfluxDBTest.setUp:58 » InfluxDBIO java.net.ConnectException: Failed to c...
  ITPutInfluxDBTest.setUp:58 » InfluxDBIO java.net.ConnectException: Failed to c...
  ITPutInfluxDBTest.setUp:58 » InfluxDBIO java.net.ConnectException: Failed to c...
  ITPutInfluxDBTest.setUp:58 » InfluxDBIO java.net.ConnectException: Failed to c...
  ITPutInfluxDBTest.setUp:58 » InfluxDBIO java.net.ConnectException: Failed to c...
  ITPutInfluxDBTest.setUp:58 » InfluxDBIO java.net.ConnectException: Failed to c...
```

Can you please let me know how I can skip running the integration tests without using this profile ?

Thanks for @MikeThomsen and your feedback.

Reply:
Oh my bad, I didn't pay close attention... Usually the IT classes do not include the word Test. That's why I thought the root profile would be enough. I'd suggest to rename the IT classes by removing the Test word to be consistent with the other bundles.